[Lutron] Fix broken dependency on gnu.io

## Expected Behavior
Install `binding-lutron - 2.4.0.M6` without problems

## Current Behavior
`binding-lutron - 2.4.0.M6` throws a FeatureInstaller ERROR when being installed on a clean OH2.4.0M6 due to missing serial bundles.

## Possible Solution
Install openhab-transport-serial prior to the installation of the lutron binding
From console:
```
feature:install openhab-transport-serial
```

## Steps to Reproduce (for Bugs)
1. Install OH2.4.0.M6
2. Use PaperUI to install `binding-lutron - 2.4.0.M6` 
3. Check log for ERROR

## Logs
```
2018-11-26 08:59:13.046 [ERROR] [core.karaf.internal.FeatureInstaller] - Failed installing 'openhab-binding-lutron': Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=openhab-binding-lutron; type=karaf.feature; version="[2.4.0.M6,2.4.0.M6]"; filter:="(&(osgi.identity=openhab-binding-lutron)(type=karaf.feature)(version>=2.4.0.M6)(version<=2.4.0.M6))" [caused by: Unable to resolve openhab-binding-lutron/2.4.0.M6: missing requirement [openhab-binding-lutron/2.4.0.M6] osgi.identity; osgi.identity=org.openhab.binding.lutron; type=osgi.bundle; version="[2.4.0.M6,2.4.0.M6]"; resolution:=mandatory [caused by: Unable to resolve org.openhab.binding.lutron/2.4.0.M6: missing requirement [org.openhab.binding.lutron/2.4.0.M6] osgi.wiring.package; filter:="(osgi.wiring.package=gnu.io)"]]
```

## Your Environment
Platform information:

Hardware: Laptop `i5-7200U@2.5GHz` / 8Gigs RAM / 256Gigs SSD
OS: Windows 10 Ent x64
Java Runtime Environment: Oracle 1.8.0_192
openHAB version: OH 2.4.0 M6

--- a/features/openhab-addons/src/main/feature/feature.xml
+++ b/features/openhab-addons/src/main/feature/feature.xml
@@ -303,6 +303,7 @@
 
     <feature name="openhab-binding-lutron" description="Lutron Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
+        <feature>openhab-transport-serial</feature>
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.lutron/${project.version}</bundle>
     </feature>
 
@@ -439,6 +440,7 @@
 
     <feature name="openhab-binding-pioneeravr" description="PioneerAVR Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
+        <feature>openhab-transport-serial</feature>
         <feature>openhab-transport-upnp</feature>
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.pioneeravr/${project.version}</bundle>
     </feature>

